packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -877,6 +877,10 @@ class MediaQuery extends InheritedWidget {
     );
   }
 
+  /// If false, device gesture settings are not read from the window and left as their
+  /// default.
+  static bool enableDeviceGestureSettings = true;
```

Comment:
Ugh. Do you have plans to clean up google3 and remove this flag in the near future?

Also, should we mark this flag es deprecated to discourage new users from using it? I assume it's only meant as a temporary measure and not a long-term API?

Reply:
Ugh is right. I am hoping I could roll, cleanup the flag, and then delete it. I tried to run a global presubmit and I only got scuba failures - so maybe I don't need it. WDUT?

Reply:
but it would be easier to remove after the fact too.

Reply:
lets give it a shot without the static

Reply:
> I am hoping I could roll, cleanup the flag, and then delete it.

Wouldn't you have to start out with the flag being false to allow the role to go in? So it would more be like roll, set it t true within google and clean up, switch the default to true, remove the flag in google, remove the flag in open source land.

Reply:
I was thinking it would be easier to g3 fix instances of the failure ... but I dunno, if its just scubas we can just accept the new diffs. I think the test that failed prior may not exist anymore